cross-chain-atomic-swap-cordapp/src/main/kotlin/jp/co/layerx/cordage/crosschainatomicswap/flow/LockEtherFlow.kt

```diff
@@ -15,20 +17,17 @@ import java.math.BigInteger
 @StartableByRPC
 class LockEtherFlow(
     private val proposalState: ProposalState,
-    private val settlement: Settlement = Settlement.load(targetContractAddress, web3, credentials, StaticGasProvider(BigInteger.valueOf(1), BigInteger.valueOf(50000000)))
+    private val settlement: Settlement?
```

Comment:
Is it possible to set a default value to `null` so that you can replace the way of call LockEtherFlow from

```
LockEtherFlow(finalizedProposalState, null)
```

to

```
LockEtherFlow(finalizedProposalState)
```

Reply:
Done